app/services/email_sender/pseudo.rb

```diff
@@ -0,0 +1,13 @@
+module EmailSender
+  class Pseudo
+    def call(address:, subject:, body:)
+      logger.info("Sending email to #{address}\r\nSubject: #{subject}\r\nBody: #{body}\r\n\r\n")
```

Comment:
Might be nicer to use a multiline string here:

```ruby
%{
Sending email to #{address}
Subject: ...
}
```

Reply:
updated that